front/themes/platinum/tailwind.config.js

```diff
@@ -0,0 +1,24 @@
+module.exports = {
+  content: [
+    '../../../apps/templates/base_public.html',
```

Comment:
I'm assuming that this is content glob is relative to the tailwind.config.js. Ideally this should be extended from a "default theme" config, so new child themes wouldn't need to re-define content each time, or they could just extend it.